tg_bot/modules/custom_messages.py

```diff
@@ -19,8 +20,18 @@ def display_lop(bot: Bot, update: Update):
 	For that please describe exactly what your bot should do and how much you are willing to pay.
 	Interested members will get in touch with you.
 	"""
-	update.effective_message.reply_text(text)
-			
+	update.effective_message.reply_text(text)		
+
+@bot_admin
+@bot_can_delete
+@user_admin
+### Deletes the sent message and repeats it	
+def say(bot:Bot, update: Update):
+	chat = update.effective_chat  # type: Optional[Chat]
+	message_id = update.effective_message.message_id
+	message = update.effective_message.text
+	bot.deleteMessage(chat.id,message_id)
+	bot.sendMessage(chat.id,message)
```

Comment:
```suggestion
	bot.sendMessage(chat.id,message[4:])
```

This is needed so that only the message itself is sent out, not the complete original message that includes the /say